### PR TITLE
Added error message for incorrect user name

### PR DESCRIPTION
**Pull request by inariDeeTee**
Separated user name check and password check into two separate methods. The method checkUserName() loops until correct user name entered and prints error message when incorrect user name entered; password check method authenticate() allows 3 attempts as before.

Also added isValidUserName method to NewBank - checks that a given string is a key in Customers and returns a boolean value.

### CODE REVIEW

**Comment by cheb2333** [comment]
Hi Heather, sorry to put you on hold here, but I would like to try and merge the other pull request first to get the major refactoring in and make sure that everyone can work on the remaining tasks without having to rework everything after the merge. I hope to get the other merge through soon, as I'm working with Shinichi to fix a slight formatting issue, and then we can continue here.

**Comment by inariDeeTee** [comment]
No problem, it should merge fine either way so it can wait while you concentrate on Shinichi's merge.

**Review by cheb2333 — APPROVED** [review]

**Pull request merged**
